vars/sendMessageWithAudit.groovy

```diff
@@ -13,6 +13,7 @@ def call(Map parameters = [:]) {
     def msgTopic = parameters.get('msgTopic')
     def msgProps = parameters.get('msgProps', '')
     def msgContent = parameters.get('msgContent', '')
+    def msgProvider = parameters.get('provider', '')
```

Comment:
Is there any reason why we cannot rely on null here?
What is a benefit of empty string as a default value?